docs/api/server.md

```diff
@@ -41,25 +43,58 @@ A restify server has the following properties on it:
 |acceptable|Array(String)|List of content-types this server can respond with.|
 |url|String|Once listen() is called, this will be filled in with where the server is running.|
 
-# Methods
+## Methods
+
+### del(opts, handler)
+### get(opts, handler)
+### head(opts, handler)
+### opts(opts, handler)
+### post(opts, handler)
+### put(opts, handler)
+### patch(opts, handler)
+
+Install a route into the restify server to handle a specific http verb.
 
-## address()
+* `opts` {String | Regex | Object}
+* `opts.name` {String} a name for the route
+* `opts.path` {String | Regex} a string or regex matching the route
+* `opts.version` {String | ArrayOfString} versions supported by this route
+* `handler` {Function | ArrayOfFunctions} a function or array of functions to
+handle this route
+
+Some examples:
+
+```js
+// a static route
+server.get('/foo', function(req, res, next) {});
+// a parameterized route
+server.get('/foo/:bar', function(req, res, next) {});
+// a regular expression
+server.get(/^\/([a-zA-Z0-9_\.~-]+)\/(.*)/, function(req, res, next) {});
+// an options object
+server.get({
+    path: '/foo',
+    version: ['1.0.0', '2.0.0]
+}, function(req, res, next) {});
+```
+
+### address()
 
 Wraps node's [address()](http://nodejs.org/docs/latest/api/net.html#net_server_address).
 
-## listen(port, [host], [callback])
+### listen(port, [host], [callback])
 
 Wraps node's [listen()](http://nodejs.org/docs/latest/api/net.html#net_server_listen_path_callback).
```

Comment:
It doesn't look like GitHub's markdown likes `'`?

Reply:
Thanks @sean3z!